packages/lib/services/plugins/RepositoryApi.ts

```diff
@@ -181,6 +181,10 @@ export default class RepositoryApi {
 					break;
 				}
 			}
+			if (manifest.id === query) {
+				output = [];
+				output.push(manifest);
+			}
```

Comment:
I don't know what this hack is for, but please remove it. If you need strict equality that would have to implemented properly and with a spec.

Reply:
https://github.com/laurent22/joplin/blob/5adee9085c2ae0b8108dce64fd8584d8777db4df/packages/lib/services/plugins/RepositoryApi.ts#L168-L187

This is the current search code, it only checks for the ```name``` and ```description``` key in the Manifest so if I give the id as a search query it won't show anything, and if I add the ```id```  to the for loop if will show all the data in the id if searched (like if Joplin is searched it will show all the plugin which contains Joplin in the id and it may not be appropriate), So ```manifest.id === query``` will only show the exact plugin if the exact search query is given and it won't interfere with any other search results.
https://github.com/laurent22/joplin/blob/5adee9085c2ae0b8108dce64fd8584d8777db4df/packages/lib/services/plugins/RepositoryApi.ts#L175